static/app/views/settings/organizationMembers/organizationMemberRow.tsx

```diff
@@ -71,6 +77,25 @@ export default class OrganizationMemberRow extends PureComponent<Props, State> {
     onSendInvite(member);
   };
 
+  renderMemberRole() {
+    const {
+      member: {flags, roleName, pending, expired},
+    } = this.props;
+    const memberDisabled = flags['member-limit:restricted'];
```

Comment:
I would probably just inline it into the if statement. There's enough context here I think to understand what it is without the variable name